packages/node_modules/@webex/plugin-meetings/src/reachability/index.js

```diff
@@ -101,12 +104,13 @@ export default class Reachability extends StatelessWebexPlugin {
             resolve({});
           }
 
-          this.peerConnectionMap = localSDPData.peerConnection;
-          ({localSDPList} = localSDPData);
-          this.setRemoteOfferForEachCluster(localSDPList)
-            .then((response) => {
-              resolve(response);
-            });
+          // TODO: Arun why did you comment this out?
```

Comment:
we currently dont need to set remote SDK so we can remove this , We are just going to track time getting the first ice